crates/connectors/ndc-postgres/src/schema.rs

```diff
@@ -626,3 +614,34 @@ fn make_procedure_type(
         },
     }
 }
+
+/// Map our local type representation to ndc-spec type representation.
+fn map_type_representation(
```

Comment:
I'm not sure we should be changing the outputted type representations like this without the user providing us an opt-in config flag to do so.

Reply:
This part is just translating our own TypeRepresentation type to ndc-spec TypeRepresentation type. Could you please elaborate on what do you mean? I'm not sure I understand.

Reply:
I was under the impression that our types should return the same `ndc-spec` `TypeRepresentation` as they did already before this change. Where this is not possible, because of removed types, we should return the more general `json`. Otherwise we're potentially changing behaviour without the user opting into it.

Reply:
What will a version of engine that does not know about these types do when it receives them?

Reply:
This is an ndc-schema concept that can be used by cli/lsp to generate engine objects, so I don't think this will be used directly in the engine. The engine already uses v0.1.2 of ndc-spec though: https://github.com/hasura/v3-engine/pull/443/files#diff-5e05c14aa4760c15d75bc66970581abe17a82eef4770a0cd9861bd40ce65934bR27

Reply:
Ahh OK, makes sense. This all seems sensible then, thank you for indulging my questions.